src/reducers/welcomeReducer.js

```diff
@@ -0,0 +1,11 @@
+import * as types from "../actions/actionTypes";
+
+export default function welcomeReducer(state = [], action) {
```

Comment:
Why did opt for the state as an array or will it change???

Reply:
Follow this [conversation](https://stackoverflow.com/questions/38445006/redux-state-as-array-of-objects-vs-object-keyed-by-id) about having the store as an array or an object and then give your point of view.

Reply:
Ok. Let me have a look.

Reply:
I have read through and have learnt that there are perfomence issues with arrays. Using objects will be a good aproach right?

Reply:
I have made changes and used an object instead of an array in the state.